### PR TITLE
Update translation_de.xml

### PR DESCRIPTION
**Pull request by Amadeo01**

### CODE REVIEW

**Comment by Amadeo01** [comment]
I have inserted the German translation. I hope it went right!

File: translations/translation_de.xml [hunk 1]
```diff
@@ -89,18 +89,18 @@
         <text name="msg_welcome" text="Willkommen zum Seasons Mod by Realismus Modding. Benötigst Du Hilfe oder eine Erklärung nutze das Hilfemenü (per ESC erreichbar)." />
 
         <!-- Missing translation of "Calendar" -->
-        <text name="ui_pageCalender" text="" />
```
Comment (joskuijpers) [inline translations/translation_de.xml]: Would something like Ekonomie be better? You cant see the prices, but more economical info: the trend of the price over a year. Different from the price table in the esc menu

**Comment by joskuijpers** [comment]
Hey :) Can you do a small update?

**Comment by Amadeo01** [comment]
I translated yesterday and inserted. Unfortunately the procedure was wrong.(??)  :-((
The addition I have previously viewed in the game under ALT links + S. It's a perfect addition!
Please enter the following five words into the translation_de:


<text name="ui_pageCalender" text="Kalender" />

<text name="ui_pageEconomy" text="Preise" />
….
….

<text name="ui_economy_animal" text="Tiere" />

<text name="ui_economy_thousand" text="1000 l" />

<text name="ui_economy_bale" text="Ballen" />

**Pull request closed without merging**